torchtune/models/llama3/_model_builders.py

```diff
@@ -62,7 +62,7 @@ def llama3_70b() -> TransformerDecoder:
         rope_base=500000.0,
     )
 
-
```

Comment:
🙃

Reply:
lol - after removing the inadvertent 405B builder, I was trying to keep the same formatting of two blank lines between each model_builder.